gallery/assets/coco/images/000000000001.jpg

```diff
@@ -0,0 +1 @@
+/home/philip/git/pytorch/vision/gallery/assets/astronaut.jpg
```

Comment:
Is that a `.jpg` file that contains text? Or is GitHub tricking me?
Is the absolute path (`/home/philip/git/...`) going to break?

Reply:
No it was a symlink to avoid duplicating the image. But I guess that didn't work out. Let me see how it should be done for git.

Reply:
Fixed with relative symlink